test/orders/orderGetAll.spec.js

```diff
@@ -0,0 +1,85 @@
+import { expect } from 'chai';
+import * as orderHelper from '../../helpers/orderHelper';
+import * as clientHelper from '../../helpers/clientHelper';
+import * as vendorHelper from '../../helpers/vendorHelper';
+import * as serviceHelper from '../../helpers/serviceHelper'
+
+describe('ORDER GET ALL', ()=>{
+  let order, orderList = [], clientId, vendorId, serviceId, resGetAll
+  before(async()=>{
+    clientId = (await clientHelper.createClient(clientHelper.clientData)).body.payload;
+    vendorId = (await vendorHelper.createVendor(vendorHelper.vendorData)).body.payload
+    serviceId = (await serviceHelper.createService(vendorId)).body.payload;
+    for(let i = 0; i < 15; i++){
+      order = await orderHelper.createOrder(orderHelper.orderData(clientId, serviceId))
+      orderList.push(order.body.payload)
+    }
+  })
+  after(async ()=>{
+    await vendorHelper.deleteVendor(vendorId);
+    await clientHelper.deleteClient(clientId);
+    await serviceHelper.deleteService(serviceId);
+    for(let i = 0; i < orderList; i++){
+      await orderHelper.deleteOrder(orderList[i])
+    }
+  })
+
+  describe('ORDER GET ALL POSITIVE', ()=>{
+    before(async()=>{
+      resGetAll = await orderHelper.orderGetAll(process.env.TOKEN)
+    })
+
+    it('verify response status', () => {
+      expect(resGetAll.statusCode).to.eq(200)
+    })
+
+    it('verify response message', () => {
+      expect(resGetAll.body.message).to.eq('OrderSearch ok')
+    })
+
+    it('verify response contains array', () => {
+      expect(resGetAll.body.payload.items).to.be.a('array');
+    });
+
+    it('verify response array has at least one element', () => {
+      expect(resGetAll.body.payload.items.length).to.be.above(0);
```

Comment:
if you have a loop with 15 iteration for order create guess better to use `to.be.at.least(15)` assertion. What do you think?